PPP-Philippines/src/dependencies/scraping/scraper.py

```diff
@@ -0,0 +1,63 @@
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+
+
+class MyClass:
```

Comment:
Change the class name to something sensible

Reply:
Pretty unclear from where the data is loaded and then saved to

Reply:
I was not saving intermediate data earlier. Made the changes to incorporate that. Also changed the names of classes.